app/services/manifest_builder.rb

```diff
@@ -1,32 +1,61 @@
 class ManifestBuilder
-  attr_reader :record
+  attr_reader :record, :light
```

Comment:
Yeah. It stops the canvases from the scanned resources being added to the document.